common/tools/coin_info.py

```diff
@@ -371,9 +371,10 @@ def _load_builtin_ethereum_networks() -> Coins:
     chains_data = load_json("ethereum", "networks.json")
     networks: Coins = []
     for chain_data in chains_data:
+        chain_id = chain_data["chain_id"]
         chain_data.update(
-            chain_id=chain_data["chain_id"],
-            key=f"eth:{chain_data['shortcut']}",
+            chain_id=chain_id,
```

Comment:
This update of `chain_data.chain_id` with the same `chain_data.chain_id` does not change anything, does it? I removed it in `definitions` repo - there is a very similar code

Reply:
good catch, yes, this looks useless. I think this is a vestige of code that changed chain_id to `str`?

Reply:
[a3bbb72](https://github.com/trezor/trezor-firmware/pull/2410/commits/a3bbb72698c10e9cd2c11c25832d47d23c5ea900)